### PR TITLE
created exception handler, added annotations to dto, controller

### PR DESCRIPTION
**Pull request by suprun-anna**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -64,6 +64,11 @@
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-web</artifactId>
 		</dependency>
+
+		<dependency>
+			<groupId>org.hibernate.validator</groupId>
+			<artifactId>hibernate-validator</artifactId>
```
Comment (Rommelua) [inline pom.xml]: better use springboot starter

File: src/main/java/mate/academy/bookstore/dto/CreateBookRequestDto.java [hunk 2]
```diff
@@ -1,14 +1,22 @@
 package mate.academy.bookstore.dto;
 
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
 import java.math.BigDecimal;
 import lombok.Data;
 
 @Data
 public class CreateBookRequestDto {
+    @NotNull
```
Comment (Rommelua) [inline src/main/java/mate/academy/bookstore/dto/CreateBookRequestDto.java]: better use NotBlank

File: src/main/java/mate/academy/bookstore/dto/CreateBookRequestDto.java [hunk 3]
```diff
@@ -1,14 +1,22 @@
...
     private String title;
+    @NotNull
     private String author;
     private String isbn;
```
Comment (Rommelua) [inline src/main/java/mate/academy/bookstore/dto/CreateBookRequestDto.java:16]: add NotBlank

File: src/main/java/mate/academy/bookstore/dto/CreateBookRequestDto.java [hunk 4]
```diff
@@ -1,14 +1,22 @@
...
+    @NotNull
+    @Min(0)
     private BigDecimal price;
+    @NotNull
     private String description;
+    @NotNull
     private String coverImage;
```
Comment (Rommelua) [inline src/main/java/mate/academy/bookstore/dto/CreateBookRequestDto.java]: is these are mandatory fields?

**Review by koval-kn — APPROVED** [review]

**Review by koval-kn — APPROVED** [review]

**Pull request merged**